network-sdk-sample/app/src/main/java/com/karhoo/samples/networksdk/booking/AdyenTripBookingFragment.kt

```diff
@@ -0,0 +1,396 @@
+package com.karhoo.samples.networksdk.booking
+
+import android.app.AlertDialog
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.Observer
+import com.adyen.checkout.base.model.PaymentMethodsApiResponse
+import com.adyen.checkout.base.model.payments.Amount
+import com.adyen.checkout.card.CardConfiguration
+import com.adyen.checkout.core.api.Environment
+import com.adyen.checkout.dropin.DropIn
+import com.adyen.checkout.dropin.DropInConfiguration
+import com.braintreepayments.api.models.PaymentMethodNonce
+import com.karhoo.samples.networksdk.R
+import com.karhoo.samples.networksdk.SampleApplication
+import com.karhoo.samples.networksdk.base.BaseFragment
+import com.karhoo.samples.networksdk.booking.AdyenResultActivity.Companion.RESULT_KEY
+import com.karhoo.samples.networksdk.planning.BookingPlanningStateViewModel
+import com.karhoo.samples.networksdk.planning.BookingStatus
+import com.karhoo.samples.networksdk.quotes.BookingQuoteStateViewModel
+import com.karhoo.samples.networksdk.quotes.QuoteListStatus
+import com.karhoo.samples.networksdk.utils.CurrencyUtils
+import com.karhoo.samples.networksdk.utils.orZero
+import com.karhoo.sdk.api.KarhooApi
+import com.karhoo.sdk.api.KarhooApi.userStore
+import com.karhoo.sdk.api.KarhooError
+import com.karhoo.sdk.api.KarhooSDKConfiguration
+import com.karhoo.sdk.api.datastore.user.SavedPaymentInfo
+import com.karhoo.sdk.api.datastore.user.UserManager
+import com.karhoo.sdk.api.model.*
+import com.karhoo.sdk.api.model.adyen.AdyenAmount
+import com.karhoo.sdk.api.network.request.AdyenPaymentMethodsRequest
+import com.karhoo.sdk.api.network.request.PassengerDetails
+import com.karhoo.sdk.api.network.request.Passengers
+import com.karhoo.sdk.api.network.request.TripBooking
+import com.karhoo.sdk.api.network.response.Resource
+import kotlinx.android.synthetic.main.fragment_trip_booking.*
+import org.json.JSONObject
+import java.util.*
+
+class AdyenTripBookingFragment : BaseFragment(), UserManager.OnUserPaymentChangedListener {
+
+    private var locale: String = "GB"
```

Comment:
Could add a constant to the companion object as DEFAULT_LOCALE

Reply:
Done